script/update

```diff
@@ -38,6 +39,10 @@ solutions = [
 ]
 {cache_dir_line}
 '''
+SRC_FILES_TO_IGNORE = [
+  'src\\native_client',
+  'src/native_client'
+]
```

Comment:
Would `os.path.join(src, 'native_client')` work here?

Reply:
Good point, I'll change it